lib/ansible/modules/cloud/azure/azure_rm_cdnendpoint.py

```diff
@@ -55,30 +55,35 @@
         suboptions:
             name:
                 description:
-                    - Origin name
+                    - Origin name.
                 required: true
             host_name:
                 description:
-                    - The address of the origin. It can be a domain name, IPv4 address, or IPv6 address.
+                    - The address of the origin.
+                    - It can be a domain name, IPv4 address, or IPv6 address.
                 required: true
             http_port:
                 description:
-                    - The value of the HTTP port. Must be between 1 and 65535
+                    - The value of the HTTP port. Must be between C(1) and C(65535).
```

Comment:
type: int 


Besides the description, the type is the most important property. Only str can be omitted